llx/builtin_ip.go

```diff
@@ -59,6 +59,23 @@ func NewIP(s string) IP {
 
 var bitmasks = []byte{0x00, 0x80, 0xc0, 0xe0, 0xf0, 0xf8, 0xfc, 0xfe, 0xff}
 
+func int2ip(i int64) string {
```

Comment:
could we use `uint32` instead for this function? we only need 32 bits and, we ipv4 addresses are naturally unsigned.

Reply:
I have a follow-up that changes this in a few moments, goes into basically any int type